### DISCUSSION TITLE
ImagePlugin

### DISCUSSION THREAD

**Discussion by mweel1** [facebook/lexical#3968] [Q&A] [answered]
Why are all the plugins in the playground not part of the library?  It seems like some are, and some aren't.

I use JS in my project and it's a bit of a pain to move these in from .tsx.

Any suggestions?

**Comment by thegreatercurve** [comment, marked as answer]
For a couple of reasons. Firstly, we, the core team, simply don't have the time to maintain the core framework and a whole host of different plugins (not forgetting our day jobs as well). Secondly, the plugins in the playground are mainly there for illustrative and E2E testing purposes. It would take a fair bit of work to make them fully production-ready. We would love OSS contributions though if any one feels like they can make a more composable and fully-featured version of any of these plugins!
